src/pgrn-list-broken-indexes.c

```diff
@@ -10,12 +13,84 @@
 
 PGDLLEXPORT PG_FUNCTION_INFO_V1(pgroonga_list_broken_indexes);
 
+static grn_ctx *ctx = &PGrnContext;
+
 typedef struct
 {
 	Relation indexes;
 	TableScanDesc scan;
 } PGrnListBrokenIndexData;
 
+static bool
+PGrnIsLockedColumn(grn_obj *table)
+{
+	grn_hash *columns;
+	bool is_locked = false;
+
+	columns = grn_hash_create(
+		ctx, NULL, sizeof(grn_id), 0, GRN_TABLE_HASH_KEY | GRN_HASH_TINY);
+	if (!columns)
+	{
+		PGrnCheck("failed to create columns container for checking locks: "
+				  "<%s>",
+				  PGrnInspectName(table));
+	}
```

Comment:
We don't need this condition because `PGrnCheck()` does nothing if `ctx->rc == GRN_SUCCESS`:

```suggestion
	PGrnCheck("failed to create columns container for checking locks: "
			  "<%s>",
			  PGrnInspectName(table));
```